### ISSUE TITLE
Easily provide the template Id to HTMLElement

### ISSUE THREAD

**Issue by GabrielBB** [WICG/webcomponents#968]
What do you guys think about providing the `templateId` via `super` `constructor`? Example:

```js
constructor {
  super({ templateId: "..." })
}
```

`HTMLElement` could then by itself get the HTML and attach it to the shadow DOM. Or it could also be a class that extends `HTMLElement` and adds this capability, therefore keeping `HTMLElement` dumb and simple. 

Probably libraries already do something like this, but hmm, I would expect basic stuff like this to be provided without libraries.

**Comment by samueldaviddelacruz** [comment]
Agree, also something like a `this.setTemplate({templateId:""})` that could be called on the constructor could be good to avoid writing so much boilerplate.
